src/components/learner-credit-management/data/utils.js

```diff
@@ -768,3 +768,10 @@ export const getAssignableCourseRuns = ({
   // Sorts by the enrollBy date. If enrollBy is equivalent, sort by start
   return assignableCourseRuns.sort(startAndEnrollBySortLogic);
 };
+
+let lastId = 0;
+
+export const newId = (prefix = 'id') => {
```

Comment:
nit: I would include this in BudgetCheckboxFilter.jsx since that is the only place that uses it, and we're likely to switch back to CheckboxFilter once it's fixed.